src/components/form/PriceInputPanel/index.tsx

```diff
@@ -26,40 +26,58 @@ interface CurrencyInputPanelProps {
   auctioningToken: Maybe<Token>
   biddingToken: Maybe<Token>
   label: string
-  onUserPriceInput: (val: string) => void
+  onUserPriceInput: (val: string, isInvertedPrice: boolean) => void
+  onInvertPrices: () => void
   value: string
+  invertPrices: boolean
 }
 
 export default function PriceInputPanel({
   auctioningToken = null,
   biddingToken = null,
+  invertPrices,
   label,
+  onInvertPrices,
   onUserPriceInput,
   value,
 }: CurrencyInputPanelProps) {
   return (
     <FormRow>
-      <FormLabel text={label} />
+      <FormLabel onInvertPrices={onInvertPrices} text={label} />
       <TextfieldWrapper>
         <NumericalInput
           onUserSellAmountInput={(val) => {
-            onUserPriceInput(val)
+            onUserPriceInput(val, invertPrices)
           }}
           value={value}
         />
         <TokenInfo>
           {auctioningToken && biddingToken ? (
-            <DoubleLogo
-              auctioningToken={{
-                address: auctioningToken.address,
-                symbol: auctioningToken.symbol,
-              }}
-              biddingToken={{
-                address: biddingToken.address,
-                symbol: biddingToken.symbol,
-              }}
-              size="24px"
-            />
+            invertPrices ? (
+              <DoubleLogo
+                auctioningToken={{
+                  address: biddingToken.address,
+                  symbol: biddingToken.symbol,
+                }}
+                biddingToken={{
+                  address: auctioningToken.address,
+                  symbol: auctioningToken.symbol,
+                }}
+                size="24px"
```

Comment:
`DoubleLogo` could be responsible for ordering them, maybe for another developer in the future, a prop called `biddingToken` receiving an `auctioningToken` can be read as an error.